src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -1,37 +1,36 @@
 import React, { useState } from 'react';
 import classNames from 'classnames';
+import { bdlBoxBlue } from 'box-ui-elements/es/styles/variables';
 import ColorPickerPalette from './ColorPickerPalette';
-import { AnnotationMode } from '../annotations/types';
+import { AnnotationColor } from '../../../AnnotationModule';
 import './ColorPickerControl.scss';
 
+const colors = Object.values(AnnotationColor);
+
 export type Props = {
-    annotationMode?: AnnotationMode;
-    onAnnotationColorClick: (color: string) => void;
+    annotationColor?: string;
     isActive?: boolean;
+    onColorSelect: (color: string) => void;
 };
 
 export default function ColorPickerControl({
-    annotationMode,
+    annotationColor = bdlBoxBlue,
```

Comment:
If this component is meant to be generic, should this be renamed `activeColor`?

Reply:
Yes that sounds good.